.github/workflows/create-release.yml

```diff
@@ -0,0 +1,58 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Upload Release Asset
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build project # This would actually build your project, using zip for an example artifact
+        run: |
+          make release
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: false
```

Comment:
@sashman what do you think about setting this to true? GH says "This is a draft and won’t be seen by the public unless it’s published." for drafts, so after we `git tag v*` and `git push --tags`, we could go to the GH release tab, add the release notes then hit the publish button. We could avoid that and Edit a published release to add the notes into the desc box. Both methods achieve the same thing but if we want to add some other hooks/actions in the future that use the desc, the former might be cleaner?

Reply:
Sounds good, there is more tooling available to create changelog notes on a release, however, this makes sense at the moment.